.github/workflows/ci.yml

```diff
@@ -2,8 +2,6 @@ name: CI
 
 on:
   push:
-    branches:
-      - main
```

Comment:
Why run on push to all branches?

Reply:
I primarily made this change for testing, so the tests would actually run on my changes before I put in a PR. I'll revert it.